.github/workflows/cd.yml

```
@@ -1,115 +1,79 @@
-            # docker-compose up이 자동으로 이미지를 pull 하지만, 명시적으로 실행하는 것이 안전합니다.
-            docker pull ${{ env.IMAGE_NAME }}
+      - name: Upload CodeDeploy bundle to S3
+        run: aws s3 cp deploy.zip s3://YOUR_CODEDEPLOY_REVISIONS_BUCKET/YOUR_APP_NAME/${{ github.sha }}.zip
 
-            # ⭐️ 4. docker compose로 컨테이너 실행
-            # 기존 컨테이너가 있다면 중지
-            docker compose down
-            docker compose up -d --force-recreate
+      - name: Start CodeDeploy deployment
+        id: start-deployment
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:40: Double quote to prevent globbing and word splitting

(shellcheck)